selfservice/strategy/passkey/.schema/settings.schema.json

```diff
@@ -7,7 +7,7 @@
       "type": "string"
     },
     "method": {
-      "const": "passkey"
+      "type": "string"
```

Comment:
I think this was correct, could you please explain why this would need to change?

Reply:
I'm quite confident that's not true, or a majority of our e2e tests would fail. I checked the slack thread but could not really find what the bug exactly is other than getting a success status without updating. If it would indeed fail here, it would fail with a validation error and not with a success state.

Reply:
OK I see in the thread now - we do apparently return a 400 error but with a success state? Not sure how that can happen

Reply:
> We "swallow" the error as part of the settings flow somewhere. Hence, the 400 with a success state of the flow?

That's quite scary and we need to investigate it.

So what is the problem exactly and why does it not happen for our e2e pipeline which includes react and nodejs?

Reply:
AFAICT, the lookup secrets implementation in the Ory Elements doesn't submit the `method` (which is incorrect, and we should fix this!). And I guess this is the main problem. 

Ideally, Kratos would require the method parameter and only execute the strategy that is sent via `method` (and only try all if no method is sent), but that would be quite a substantial change.